dropwizard-lifecycle/src/test/java/io/dropwizard/lifecycle/setup/ScheduledExecutorServiceBuilderTest.java

```diff
@@ -0,0 +1,142 @@
+package io.dropwizard.lifecycle.setup;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertSame;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+import java.util.concurrent.ThreadFactory;
+
+import org.junit.After;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+
+import io.dropwizard.lifecycle.ExecutorServiceManager;
+import io.dropwizard.util.Duration;
+
+public class ScheduledExecutorServiceBuilderTest {
+
+    private static final Duration DEFAULT_SHUTDOWN_PERIOD = Duration.seconds(5L);
+
+    private ScheduledExecutorService execTracker;
+
+    public ScheduledExecutorServiceBuilderTest() {
+        super();
+        this.execTracker = null;
+    }
+
+    @After
+    public void tearDown() {
+        if (this.execTracker != null) {
+            this.execTracker.shutdown();
+            this.execTracker = null;
+        }
+    }
+
+    @Test
+    public void testBasicInvocation() {
+        final LifecycleEnvironment le = mock(LifecycleEnvironment.class);
+
+        final String poolName = this.getClass().getSimpleName();
+
+        final ScheduledExecutorServiceBuilder test = new ScheduledExecutorServiceBuilder(le,
+            poolName,
+            false);
+
+        this.execTracker = test.build();
+        assertTrue(this.execTracker instanceof ScheduledThreadPoolExecutor);
```

Comment:
We use AssertJ for assertions in new unit tests. It would be great to rewrite the assertions in the "fluent" AssetJ style, because that's way they are more readable and IDE-friendly.